docs/docsite/sphinx_conf/2.10_conf.py

```diff
@@ -43,7 +43,7 @@
 # They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 # TEST: 'sphinxcontrib.fulltoc'
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'notfound.extension']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'notfound.extension', 'sphinx_antsibull_ext']
```

Comment:
How about making this multiline and adding a comment on why the last ext is needed?

Reply:
Done.